src/gui/sharepermissions.h

```diff
@@ -23,12 +23,12 @@ namespace OCC {
  * Possible permissions, must match the server permission constants
  */
 enum SharePermission {
-    SharePermissionRead     = 1 << 0,
-    SharePermissionUpdate   = 1 << 1,
-    SharePermissionCreate   = 1 << 2,
-    SharePermissionDelete   = 1 << 3,
-    SharePermissionShare    = 1 << 4,
-    SharePermissionDefault  = 1 << 30
```

Comment:
@claucambra Could you elaborate? I am not sure I understand the change

Reply:
`SharePermissionDefault` is not really a default value but rather an "all" value, hence it having the value 31. in a bit field this is 11111 . The previous value was a gigantic int that was not 31